sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1493,15 +1493,14 @@ object SQLConf {
 
   val CAN_CHANGE_CACHED_PLAN_OUTPUT_PARTITIONING =
     buildConf("spark.sql.optimizer.canChangeCachedPlanOutputPartitioning")
-      .internal()
```

Comment:
BTW, you don't need to expose this. As you see in the most legacy configs, `.internal()` is fine.